OpenTrons/opentrons-api
# refactor(protocol-designer): a few style fixes with modal size and alert text

closes RQA-2520 RQA-2521 RQA-2525 RQA-2523 RQA-2528

# Overview

Addresses a few style things. See tickets for more details

# Test Plan

- create an ot-2 protocol and on the pipette tips tile in the create file wizard, click to add a custom labware and see that the empty state's borders is 1rem (it got changed to 200px by accident from one of Jamey's prs i think)
- create the protocol and edit the pipettes, see that the modal doesn't extend the whole height of the viewport
- add a magnetic module to the deck. add another module to be on top of that module's slot and see the module collision warning. See that it is styled to the right side 
- add a magnetic module step and see that the text is all translated correctly in i18n.

# Changelog

- revert border
- change the slot collision warning stuff
- add a max height to modal
- fix i18n translation strings
- fix the height of the off-deck slideout empty state 

# Review requests

see test plan

# Risk assessment

low
